From ff5881f1f0b0188c12daafc66ce46f49552b523d Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Tue, 19 Aug 2014 15:28:00 -0400
Subject: [tools] perf: Remove trivial extra semincolon

Message-id: <1408462094-14194-312-git-send-email-jolsa@redhat.com>
Patchwork-id: 88299
O-Subject: [PATCH RHEL7.1 BZ1131394 311/325] perf tools: Remove trivial extra semincolon
Bugzilla: 1131394
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1131394
https://bugzilla.redhat.com/show_bug.cgi?id=1131394

upstream
========
commit b222213936ef7d48908be2fab7639dd535c88045
Author: Davidlohr Bueso <davidlohr@hp.com>
Date: Tue Nov 12 22:24:24 2013 -0800

description
===========
Accidentally ran into these, get rid of them.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/ui/browser.c b/tools/perf/ui/browser.c
index bbc782e..3648d4e 100644
--- a/tools/perf/ui/browser.c
+++ b/tools/perf/ui/browser.c
@@ -680,7 +680,7 @@ static void __ui_browser__line_arrow_down(struct ui_browser *browser,
  if (end >= browser->top_idx + browser->height)
   end_row = browser->height - 1;
  else
-  end_row = end - browser->top_idx;;
+  end_row = end - browser->top_idx;
 
  ui_browser__gotorc(browser, row, column);
  SLsmg_draw_vline(end_row - row + 1);
diff --git a/tools/perf/util/evlist.c b/tools/perf/util/evlist.c
index dc6fa3f..5ce2ace 100644
--- a/tools/perf/util/evlist.c
+++ b/tools/perf/util/evlist.c
@@ -1148,7 +1148,7 @@ size_t perf_evlist__fprintf(struct perf_evlist *evlist, FILE *fp)
        perf_evsel__name(evsel));
  }
 
- return printed + fprintf(fp, "\n");;
+ return printed + fprintf(fp, "\n");
 }
 
 int perf_evlist__strerror_tp(struct perf_evlist *evlist __maybe_unused,
-- 
1.7.1